articles/quickstarts/install-command-line.md

```diff
@@ -18,7 +18,11 @@ Q# programs can be executed on their own, without a driver in a host language li
 
 ## Installation
 
-While you can build Q# command line applications in any IDE, we recommend using Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+While you can build Q# command line applications in any IDE, we recommend using Visual Studio Codespaces, Visual Studio Code (VS Code) or Visual Studio IDE for your Q# applications. Developing in these environments includes the rich functionality of the QDK extension, which includes warnings, syntax highlighting, project templates, and more.
+
+To configure Visual Studio Codespaces:
+1. Create an Azure account [here](https://azure.microsoft.com/en-us/free/).
```

Comment:
I'd suggest dropping en-us for localization, and modifying the link text for SEO.
```suggestion
1. Create an [Azure account](https://azure.microsoft.com/free/).
```